From 052732a9fb44ab63e522a9b2949d64e2d2f2d4fc Mon Sep 17 00:00:00 2001
From: Emil Renner Berthing <kernel@esmil.dk>
Date: Sat, 20 Nov 2021 19:29:25 +0100
Subject: [PATCH 1006/1021] dt-bindings: reset: Add StarFive JH7100 audio reset
 definitions

Add all resets for the StarFive JH7100 audio reset controller.

Signed-off-by: Emil Renner Berthing <kernel@esmil.dk>
---
 .../dt-bindings/reset/starfive-jh7100-audio.h | 31 +++++++++++++++++++
 1 file changed, 31 insertions(+)
 create mode 100644 include/dt-bindings/reset/starfive-jh7100-audio.h

--- /dev/null
+++ b/include/dt-bindings/reset/starfive-jh7100-audio.h
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: GPL-2.0 OR MIT */
+/*
+ * Copyright (C) 2021 Emil Renner Berthing
+ */
+
+#ifndef __DT_BINDINGS_RESET_STARFIVE_JH7100_AUDIO_H__
+#define __DT_BINDINGS_RESET_STARFIVE_JH7100_AUDIO_H__
+
+#define JH7100_AUDRSTN_APB_BUS		0
+#define JH7100_AUDRSTN_I2SADC_APB	1
+#define JH7100_AUDRSTN_I2SADC_SRST	2
+#define JH7100_AUDRSTN_PDM_APB		3
+#define JH7100_AUDRSTN_I2SVAD_APB	4
+#define JH7100_AUDRSTN_I2SVAD_SRST	5
+#define JH7100_AUDRSTN_SPDIF_APB	6
+#define JH7100_AUDRSTN_PWMDAC_APB	7
+#define JH7100_AUDRSTN_I2SDAC_APB	8
+#define JH7100_AUDRSTN_I2SDAC_SRST	9
+#define JH7100_AUDRSTN_I2S1_APB		10
+#define JH7100_AUDRSTN_I2S1_SRST	11
+#define JH7100_AUDRSTN_I2SDAC16K_APB	12
+#define JH7100_AUDRSTN_I2SDAC16K_SRST	13
+#define JH7100_AUDRSTN_DMA1P_AHB	14
+#define JH7100_AUDRSTN_USB_APB		15
+#define JH7100_AUDRST_USB_AXI		16
+#define JH7100_AUDRST_USB_PWRUP_RST_N	17
+#define JH7100_AUDRST_USB_PONRST	18
+
+#define JH7100_AUDRSTN_END		19
+
+#endif /* __DT_BINDINGS_RESET_STARFIVE_JH7100_AUDIO_H__ */
